config-sample.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Should we update this file to "config.json", or is the intent to allow the user to change it at will?

Reply:
Ah, looks like you did.